dazel.py

```diff
@@ -436,6 +439,23 @@ def _add_ports(self, ports):
         # calculate the ports string
         self.ports = '-p "%s"' % '" -p "'.join(ports)
 
+    def _add_gpus(self, gpus):
+        """Add the given ports to the run string."""
+        # This can only be intentional in code, so disregard.
+        self.gpus = ""
+        if not gpus:
+            return
+
+        # DAZEL_PORTS can be a python iterable or a comma-separated string.
+        if isinstance(gpus, str):
+            gpus = [g.strip() for g in gpus.split(",")]
```

Comment:
you're splitting by `,` here just to later join by it again. Maybe only allow strings here.

Reply:
Right, it does sanitize the string though, since it also removes spaces.

Plus it's consistent with other flags, which all allow lists of strings or iterables.

Reply:
Ok, makes sense.